components/FormAddNurse/FormAddNurse.js

```diff
@@ -124,7 +124,7 @@ export default function FormAddNurse({ onSubmitNurse }) {
         />
 
         <Label htmlFor="description">Notes</Label>
-        <Input as="textarea" name="description"></Input>
+        <Input as="textarea" name="description" required></Input>
```

Comment:
If you have input fields where you have a restriction - then you can also tell the user. Like for example - age not older than 65.

Reply:
In this case the "User" must know this in advance, as he is the "Team Leader" of the team. But I will complement later with your suggestion, as well as other warnings and alerts for the user.